code/modules/siphon/siphon_minerals.dm

```diff
@@ -0,0 +1,291 @@
+		"Type-AX Resonator, Position F4, 1 Intensity<br>"
+	)
+
+/datum/siphon_mineral/claretine
+	name = "Claretine"
+	tick_req = 40
+	x_torque = 32
+	y_torque = -4
+	shear = 20
+	sens_window = 4
+	product = /obj/item/raw_material/claretine
+	setup_guide = list(
+		"Type-AX Resonator, Position C2, 1 Intensity<br>",
+		"Type-AX Resonator, Position E2, 1 Intensity<br>",
+		"Type-AX Resonator, Position F4, 4 Intensity<br>",
+		"Type-AX Resonator, Position G6, 1 Intensity<br>"
+	)
+
+/datum/siphon_mineral/bohrum
+	name = "Bohrum"
+	tick_req = 40
+	x_torque = -16
+	y_torque = -16
+	shear = 24
+	sens_window = 4
+	product = /obj/item/raw_material/bohrum
+	setup_guide = list(
+		"Type-AX Resonator, Position C2, 2 Intensity<br>",
+		"Type-AX Resonator, Position D3, 2 Intensity<br>",
+		"Type-AX Resonator, Position G6, 1 Intensity<br>",
+		"Type-AX Resonator, Position H7, 1 Intensity<br>"
+	)
+
+/datum/siphon_mineral/fibrilith
+	name = "Fibrilith"
+	x_torque = 0
+	y_torque = 0
+	shear = 12
+	sens_window = 1
+	product = /obj/item/raw_material/fibrilith
+
+	New()
+		src.shear = rand(8,16)
+		..()
+
+/datum/siphon_mineral/cobryl
+	name = "Cobryl"
+	x_torque = -96
+	shear = 4
+	sens_window = 9
+	product = /obj/item/raw_material/cobryl
+
+/datum/siphon_mineral/syreline
+	name = "Syreline"
+	tick_req = 50
+	x_torque = 88
+	shear = 6
+	sens_window = 1
+	product = /obj/item/raw_material/syreline
+
+/datum/siphon_mineral/erebite
+	name = "Erebite"
+	tick_req = 90
+	x_torque = 6
+	y_torque = -22
+	shear = 33
+	sens_window = 2
+	product = /obj/item/raw_material/erebite
+
+	New()
+		src.shear = rand(30,40)
+		..()
+
+/datum/siphon_mineral/cerenkite
+	name = "Cerenkite"
+	tick_req = 60
+	x_torque = -24
+	y_torque = 8
+	shear = 16
+	sens_window = 3
+	product = /obj/item/raw_material/cerenkite
+
+	New()
+		src.shear = rand(8,24)
+		..()
+
+/datum/siphon_mineral/plasmastone
+	name = "Plasmastone"
+	tick_req = 70
+	x_torque = -16
+	y_torque = 13
+	shear = 4
+	sens_window = 1
+	product = /obj/item/raw_material/plasmastone
+
+	New()
+		src.shear = rand(4,10)
+		..()
+
+/datum/siphon_mineral/koshmarite
+	name = "Koshmarite"
+	tick_req = 40
+	shear = 58
+	product = /obj/item/raw_material/eldritch
+
+	New()
+		src.shear = rand(57,60)
+		..()
+
+/datum/siphon_mineral/gemstone
+	name = "Gemstone"
+	tick_req = 150
+	x_torque = 0
+	y_torque = 0
+	shear = 64
+	product = /obj/item/raw_material/gemstone
+
+/datum/siphon_mineral/uqill
+	name = "Uqill"
+	tick_req = 190
+	shear = 54
+	sens_window = 2
+	product = /obj/item/raw_material/uqill
+//telecrystal and gnesis have unusual formative conditions difficult to induce manually
+//should probably have highly specific parameters, maybe obtained through secrets?
+//idea in particular: the required shear is in the 96-105 range and chooses a different value every 60 or 90 sec
+//and the recipe is learned through a device that shows you when this value changes, so you need a rapidly adjustable resonator setup
+/*
```

Comment:
why comment

Reply:
Telecrystal synthesis would have significant implications in the Lore(TM), so it's not initially enabled; comments are a musing on the specifics of how it might happen.